app/renderer/components/window.js

```diff
@@ -93,11 +96,14 @@ class Window extends React.Component {
 
   componentDidMount () {
     appActions.windowReady(getCurrentWindowId())
+    this.updateComputedStyles()
+    window.addEventListener('resize', this.updateComputedStyles)
```

Comment:
we already have a window resize handler in windows.js that triggers `appActions.windowUpdated` and it is debounced to prevent a flood of actions during resize. I think it would be better to send that action to the window using queryInfo (see `urlBarTextChanged` for an example) and update the computed styles in response to that